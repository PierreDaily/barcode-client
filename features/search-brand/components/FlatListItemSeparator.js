import React from "react";
import { View } from "react-native";
import styles from "../styles";

const FlatListItemSeparator = () => {
  return (
    <View style={{ flex: 1, alignItems: "center" }}>
      <View style={styles.separator} />
    </View>
  );
};

export { FlatListItemSeparator };
