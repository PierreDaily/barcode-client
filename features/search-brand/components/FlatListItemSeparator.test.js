import React from "react";
import { FlatListItemSeparator } from "./FlatListItemSeparator";

import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<FlatListItemSeparator />).toJSON();
  expect(tree).toMatchSnapshot();
});
