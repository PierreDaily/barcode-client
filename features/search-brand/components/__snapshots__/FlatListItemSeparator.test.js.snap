// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#607D8B",
        "height": 1,
        "opacity": 0.3,
        "width": "92%",
      }
    }
  />
</View>
`;
